server_admin/topics/users/ref-user-credentials.adoc

```diff
@@ -27,8 +33,12 @@ Data::
    credential.
 
 Actions::
+ifeval::[{project_community}==true]
+   This column has two actions. Click *Reset passwrod* to change the passord for the user and *Delete* to remove the credential.
+endif::[]
```

Comment:
This column is not in new version.  You can remove this.
There is a Vertical ellipsis with Delete, but I think they can figure that out.